toolbox/fdc3-compliance/src/test/fdc3.findIntent.ts

```diff
@@ -0,0 +1,13 @@
+import { ResolveError } from "@finos/fdc3";
+
+describe("fdc3.findIntent", () => {
+  it("Method is callable", async () => {
+    try {
+      await window.fdc3.findIntent("ThisIntentDoesNotExist");
```

Comment:
We expect an error to be thrown so we should throw an error immediately after line 6 to make sure the test fails if no error is thrown by `findIntent()`. This would also require a rework inside of the catch statement.